test.sh

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
+set -e
+
+function usage() {
+  echo "Usage: $0 VARIANT"
+  echo "  VARIANT is the type of django-bootstrap image to test"
+}
+
+VARIANT="$1"
+shift || { usage >&2; exit 1; }
+trap '{ set +x; echo; echo FAILED; echo; } >&2' ERR
+
+set -x
+
+# Build example project image, run and wait for it to start
+docker build --tag mysite --file "example/$VARIANT.dockerfile" example
+docker run --detach --name mysite -p 8000:8000 mysite
+trap "docker stop mysite; docker rm -f mysite; docker rmi -f mysite" EXIT
+sleep 5
+
+# Simple check to see if the site is up
+curl -fsL http://localhost:8000/admin | fgrep '<title>Log in | Django site admin</title>'
```

Comment:
Yeah that's fine... the only paths we special case at the moment are `/static/` and `/media/`, everything else is just sent to gunicorn.